webbpsf/opds.py

```diff
@@ -1924,10 +1935,24 @@ def move_seg_local(self, segment, xtilt=0.0, ytilt=0.0, clocking=0.0, rot_unit='
         if not delay_update:
             self.update_opd(display=display)
 
-    def move_seg_global(self, segment, xtilt=0.0, ytilt=0.0, clocking=0.0, rot_unit='urad',
-                        radial=None, xtrans=0.0, ytrans=0.0, piston=0.0, roc=0.0, trans_unit='micron', display=False,
-                        delay_update=False, absolute=False):
-        """ Move a segment in pose and/or ROC, using PM global V coordinates..
+    def move_seg_global(
+        self,
+        segment,
+        xtilt=0.0,
+        ytilt=0.0,
+        clocking=0.0,
+        rot_unit="urad",
+        radial=None,
+        xtrans=0.0,
+        ytrans=0.0,
+        piston=0.0,
+        roc=0.0,
+        trans_unit="micron",
+        display=False,
+        delay_update=False,
+        absolute=False,
+    ):
+        """Move a segment in pose and/or ROC, using PM global V coordinates..
```

Comment:
And again this is inconsistent. This one gets expanded out to many lines, but the very next function is not.